homeassistant/components/alarm_control_panel/simplisafe.py

```diff
@@ -33,41 +33,42 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the SimpliSafe platform."""
+    from simplipy.api import SimpliSafeApiInterface, get_systems
     name = config.get(CONF_NAME)
     code = config.get(CONF_CODE)
     username = config.get(CONF_USERNAME)
     password = config.get(CONF_PASSWORD)
 
-    add_devices([SimpliSafeAlarm(name, username, password, code)])
+    simplisafe = SimpliSafeApiInterface()
+    simplisafe.set_credintials(username, password)
```

Comment:
Probably, might have to add something to the library. It is currently logged in the library when they are invalid so I should be able to return some sort of indicator back to homeassistant.